python/private/pypi/whl_library.bzl

```diff
@@ -55,9 +55,31 @@ def _get_xcode_location_cflags(rctx):
         # This is a full xcode installation somewhere like /Applications/Xcode13.0.app/Contents/Developer
         # so we need to change the path to to the macos specific tools which are in a different relative
         # path than xcode installed command line tools.
-        xcode_root = "{}/Platforms/MacOSX.platform/Developer".format(xcode_root)
+        xcode_sdks_json = rctx.execute([
```

Comment:
Please use `repo_utils.execute_checked` from `//python/private:repo_utils.bzl. We have wrappers that do some book keeping to aid debugging etc. (See a few lines above for example)